tests/data/makeTestExposure.py

```diff
@@ -0,0 +1,68 @@
+"""Create a Exposure object for testing with non-zero elements in most fields.
+"""
+import numpy as np
+
+from lsst.afw.cameraGeom.testUtils import DetectorWrapper
+import lsst.afw.image
+from lsst.afw.image.utils import resetFilters, defineFilter
+import lsst.afw.geom
+import lsst.daf.base
+import lsst.geom
+
+
+nx = 10
+ny = 10
+exp = lsst.afw.image.ExposureF(nx, ny)
+
+# Fill the maskedImage
+exp.maskedImage.image.array = np.arange(nx*ny, dtype='f').reshape(nx, ny)
+exp.maskedImage.variance.array = np.ones((nx, ny), dtype='f')
+exp.maskedImage.mask.array[5, 5] = 5  # make one pixel non-zero
```

Comment:
What you have here is not correct, but e.g. `exp.image.array` works, too.

Reply:
"is not correct": did you mean "is not incorrect"?

Reply:
Yikes, sorry, yes, that's what I meant; fixed.